AutocompleteClient/FW/Config/AutocompleteConfig.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Constants.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+public struct AutocompleteConfig {
```

Comment:
I think we can remove this struct as it duplicates the ConstructorIO config?

Reply:
Sure. This file isn't even in the project, but was picked up by git. Something probably happened during rebase.